frontend/src/constants/tabs.tsx

```diff
@@ -0,0 +1,24 @@
+import { AccountCircle, ListAlt, QrCode } from "@mui/icons-material";
+
+import { TabItem } from "@/type/common";
+
+export const Tabs: TabItem[] = [
+  {
+    key: "home",
+    name: "ホーム",
+    icon: <AccountCircle />,
+    href: "/",
+  },
+  {
+    key: "personalInfo",
+    name: "個人情報",
+    icon: <ListAlt />,
+    href: "/personal-info",
+  },
+  {
+    key: "qrcode",
+    name: "学祭QRコード",
+    icon: <QrCode />,
+    href: "/qrcode",
+  },
+];
```

Comment:
[note] UPPER_CASEを用いることでconstantsだと強調できます

[very small note] 
できれば`as const`型アサーションを使うことで、リテラル型であることを強調したいですが、TabItemとしての形を保守したい場合は現状のままで大丈夫です
参考：https://zenn.dev/yumemi_inc/articles/js-front-constants-a1fb3c49eb1199#%E3%81%93%E3%81%AE%E8%A8%98%E4%BA%8B%E3%81%A7%E6%B1%82%E3%82%81%E3%82%8B%E3%80%8C%E4%BF%9D%E5%AE%88%E6%80%A7%E3%80%8D
参考：https://typescriptbook.jp/reference/values-types-variables/type-assertion-as

```suggestion
export const TABS = [
  {
    key: "home",
    name: "ホーム",
    icon: <AccountCircle />,
    href: "/",
  },
  {
    key: "personalInfo",
    name: "個人情報",
    icon: <ListAlt />,
    href: "/personal-info",
  },
  {
    key: "qrcode",
    name: "学祭QRコード",
    icon: <QrCode />,
    href: "/qrcode",
  },
] as const;
```

Reply:
これ最初やろうとしてたんだけど、ESLintに怒られるみたいなのでやめた
![image](https://github.com/NUTFes/NUTFES-Account/assets/46074208/ad9ed6fa-8147-497e-96ce-e9c698b8df72)

Reply:
僕の環境だとESLintのエラーも出ずちゃんと型推論されるんですが、エラーが出るなら`as const`は使わなくていいと思います
リロードしたら治りそう

Reply:
リロードしたら治った....